### PR TITLE
Hemoglobin, anemia, and iron deficiency

### PR DESCRIPTION
**Pull request by alibow**
Hi all --

This PR is for a whole section in the GBD 2019 documentation folder for hemoglobin/anemia/iron deficiency related model documentation. I am thinking that it deserved its own folder because these models are so interrelated and it would have been a giant document if I tried to put it all in a risk exposure doc.

Anyway, four PRs will be following this one with the specific content for each of these pages initiated in this PR, so keep an eye out.

Thanks :)

### CODE REVIEW

File: docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+The hemoglobin model in GBD 2019 is the starting point for both the anemia impairment and iron deficiency risk factor models in GBD 2019. The hemoglobin model estimates the *distribution* of population hemoglobin concentration for each location-, age-, sex-, and year-specific demographic group. This is done by estimation of mean and standard deviation hemoglobin values and a corresponding ensemble distribution fit. 
+
+This model will allow the assignment of specific hemoglobin concentration values to indiidual simulants and will likely need to be used in order to model either the anemia impairment or iron deficiency risk factor.
+
+:ref:`The 2019 GBD Hemoglobin Model Documentation Page can be found here <2019_hemoglobin_model>`.
+
+Anemia Impaiment
+^^^^^^^^^^^^^^^^
+
+The anemia impairment model in GBD 2019 uses the hemoglobin distribution model to evaluate the total prevalence of anemia ("anemia envelope") based on age- and sex-specific thresholds. The anemia impairment model also performs *causal attribution* in which specific causes of anemia (for example, dietary iron deficiency or maternal hemorrhage) are assigned to all of the cases in the anemia envelope.
```
Comment (NathanielBlairStahn) [inline docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst:55]: "are assigned to all of the cases in the anemia envelope."

Is this supposed to say cases or causes?

File: docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+=======================================
+Hemoglobin, Anemia, and Iron Deficiency
+=======================================
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   *
+   */index
```
Comment (NathanielBlairStahn) [inline docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst:12]: This structure is slightly confusing to me, as I usually expect to see the contents for the current page at the top (i.e. with a `.. contents:: :local:` directive), but I think it's also useful to have the links to all the subpages in one place near the top.  Maybe put the local contents at the top and put this `toctree` at the end of the Topic Overview section?

**Review by NathanielBlairStahn — APPROVED** [review]
I left a couple minor comments/suggestions.  Also, I think it's a little confusing having the section headings appear at the same level as the subpages in the main table of contents under GBD 2019 Models, but I can't think of a better way to structure it given the constraints of Sphinx...

File: docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst (hunk 1 above)
Comment (alibow) [reply docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst:55]: Cases. As in a cause is assigned to each "case" of anemia, where case is individuals with anemia. I can add some clarity there.

**Review by aflaxman — APPROVED** [review]

**Pull request merged**